### PR TITLE
Split up tests, run in parallel, prefer java NIO

### PR DESCRIPTION
**Pull request by christopherbryanboyd**

### CODE REVIEW

File: cli/build.gradle [hunk 1]
```diff
@@ -169,8 +170,11 @@ task unzipPortal(dependsOn: downloadPortal, type: Copy) {
 test {
 	dependsOn createDepsZip, createWrapperZip, unzipPortal
 
-	systemProperty 'buildDir', "${buildDir}"
-	systemProperty 'liferay.home', "${buildDir}/liferay-ce-portal-7.0-ga4"
+	doFirst {
```
Comment (gamerson) [inline cli/build.gradle]: Do these need to be in a doFirst block?

File: cli/src/main/java/com/liferay/blade/cli/BladeCLI.java [hunk 2]
```diff
@@ -50,8 +50,6 @@
  */
 public class BladeCLI implements Runnable {
 
-	public static final File USER_HOME_DIR = new File(System.getProperty("user.home"));
```
Comment (gamerson) [inline cli/src/main/java/com/liferay/blade/cli/BladeCLI.java]: I don't want to include these changes just yet, I think I am going to refactor how blade gets its extensions directory, I want to change the information flow a bit.  Can you remove the changes to the userHomeDir from this PR and just send the parallel tests changes in a single PR?

File: cli/src/test/java/com/liferay/blade/cli/TestUtil.java [hunk 3]
```diff
@@ -39,6 +48,44 @@
  */
 public class TestUtil {
 
+	public static File checkFileDoesNotExists(Path path) {
```
Comment (gamerson) [inline cli/src/test/java/com/liferay/blade/cli/TestUtil.java]: none of these tests are used outside of CreateCommandTEst.  So there is no reason yet to externalize them. Lets leave them in CreateCommandTest.

File: cli/src/test/java/com/liferay/blade/cli/command/CreateCommandTest.java [hunk 4]
```diff
@@ -1,1601 +0,0 @@
-/**
```
Comment (gamerson) [inline cli/src/test/java/com/liferay/blade/cli/command/CreateCommandTest.java]: Don't move this to a new package, just leave it in same package.

File: cli/src/test/java/com/liferay/blade/cli/command/InitCommandTest.java [hunk 5]
```diff
@@ -1,442 +0,0 @@
-/**
```
Comment (gamerson) [inline cli/src/test/java/com/liferay/blade/cli/command/InitCommandTest.java]: Don't move this to new package.

**Comment by gamerson** [comment]
Closing, see comments and please resend just parallel test execution changes for now.

**Comment by christopherbryanboyd** [comment]
Hey @gamerson , here are the updates necessary only for Parallel Test Execution: https://github.com/gamerson/liferay-blade-cli/pull/450

**Pull request closed without merging**